src/templates/newengland-organizing.html

```diff
@@ -0,0 +1,102 @@
+
+                        <li><em>A Financial Report must be submitted after the competition.</em></li>
+                    </ul>
+                </blockquote>
+                <p>– <em><a href="https://cubingusa.org/supported">CubingUSA</a></em></p>
+                <p>The main benefit of this is that you can use their nonprofit status (which can result in venue discounts) and that finances are handled in a more official way. If this is something you’re interested in, let us and we can work to make this happen. Note that this doesn’t change what the competition is actually like the day of.</p>
+                <h3 id="coordinating-staff">Coordinating Staff</h3>
+                <p>We have a template of a Google Form we use to collect staff applications which the Delegate will set up. While we’re happy to provide guidance, the organizer is responsible for providing lunch to the staff and properly coordinating with all staff members.</p>
+                <p>Staff typically have perks of free registration and free lunch in exchange for helping out all day. Staff must pay to register and will be refunded after the competition, as an incentive to actually do what is expected of them. Staff applications are posted on the website to allow anyone to fill out a form and apply for staff, but it’s up to our discretion to vet these people and make sure qualified people will be running the competition.</p>
+                <p>Additionally, at recent competitions, we have been doing what is known as the “Hungarian System” where all competitors are assigned judging roles. This helps people become familiar with judging, and reduces the need for asking for volunteers constantly. It’s great to have a competitor tutorial at the beginning of the competition to help people understand how judging works, and even how competing works. This significantly reduces incidents during the competition when people are aware of the basics, like +2 penalties</p>
+                <h3 id="the-day-of">The day of</h3>
+                <p>The Delegate will bring equipment to the competition, but you should budget for battery money as the equipment requires lots of batteries. The Delegate will usually arrive early to help set up, but it’s ideal if the venue could be set up the night before. Note that the organizer is expected to be there well before the start of the competition. The organizer should also be the one making announcements, laying out scorecards, and making sure everything that needs to be happening is happening. The Delegate’s role is to ensure that WCA Regulations are being followed, not to run the event. That being said, your Delegate has done plenty of organizing before and should serve as a resource to the organizer.</p>
```

Comment:
Expand on the responsibilities of the organizer during the competition, maybe a few bullet points rather than just a paragraph list.  This is often neglected by new organizers.